src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceNotesViewModelMapTransfer.cs

```diff
@@ -3,23 +3,28 @@
     using System;
     using EA.Weee.Core.AatfEvidence;
     using System.Collections.Generic;
+    using CuttingEdge.Conditions;
     using Weee.Requests.Scheme;
 
     public class TransferEvidenceNotesViewModelMapTransfer
     {
-        public List<EvidenceNoteData> Notes { get; private set; }
+        public IList<EvidenceNoteData> Notes { get; private set; }
 
         public TransferEvidenceNoteRequest Request { get; private set; }
 
         public Guid OrganisationId { get; private set; }
 
-        public TransferEvidenceNotesViewModelMapTransfer(List<EvidenceNoteData> notes,
+        public TransferEvidenceNotesViewModelMapTransfer(IList<EvidenceNoteData> notes,
             TransferEvidenceNoteRequest transferRequest,
-            Guid organisation)
+            Guid organisationId)
         {
+            Condition.Requires(notes).IsNotNull();
```

Comment:
Can you add IsNotEmpty() too?

Reply:
Have left this as can be empty at the moment as in theory the user could filter out all the results when we get to that bit.

Reply:
okay